src/modules/main/types/experts.ts

```diff
@@ -0,0 +1,39 @@
+export interface IExperts {
+    userId: number,
+    emailId: string,
+    password: string,
+    status: ExpertStatus,
```

Comment:
that's cool,
pls check how enum should look like
https://github.com/ita-social-projects/dokazovi-fe/blob/develop/src/modules/direction/store/actionTypes.ts#L1